### DISCUSSION TITLE
Fresh installation error

### DISCUSSION THREAD

**Discussion by y0d4a** [Security-Onion-Solutions/securityonion#3188] [Unsupported Versions] [answered]
Dear,

i just try to install fresh installation of SO2 and cannot pass setup...
always get
` [ERROR   ] The Salt Master has cached the public key for this node, this salt minion will wait for 10 seconds before attempting to re-authenticate`

seems on this stage
![image](https://user-images.githubusercontent.com/3115997/109503039-11f8c380-7a9a-11eb-944b-0edcda7c0dba.png)

**Comment by AndriiStepovyi** [comment]
Please provide your setup description (Host OS, Hypervisor) and your hardware settings assigned for this installation: CPU, RAM, Storage space versus your Host capacity.
New version is known to be very demanding on requirements and drops pre-install or install in various ways if those aren't met.
I have found that Storage less then 100Gb is Major reason for it, as well as RAM lesser then 5 Gb and even then tricky. Should say for installation purposes you may assign all the required settings and readjust them later (except storage) in case of VM setup.
Let me know.

**Reply by y0d4a** [reply]
well, that shouldn't be the problem...
SO image, bare metal.
24 cores
95GB
7TB space

**Reply by AndriiStepovyi** [reply]
Sure you have capacity, but you didn't answer the question: Which Hypervisor and how much assigned for the SO installation? Have you checked the ISO integrity after obtaining vs source?

**Reply by y0d4a** [reply]
no hypervisor, full bare metal server
and yes checked and on second server (where is sensor, setup pass fine)

**Reply by AndriiStepovyi** [reply]
Got you- just SO CentOS based tenant.
If you installed via ISO- try Network install and opposite.
At this point since no Salt-Master and Minions yet up and running to mingle with files, your only option is to try different reinstall I guess. So far those who had issues with ISO and used Network one- had passed the ERROR point.

**Reply by y0d4a** [reply]
i tried with network also...
after first fail, i switch to network setup, but without success :/

**Reply by y0d4a** [reply]
update, i try to do update to latest ver.
and got this error:


```
This is a manager, We can proceed.
Checking to see if this is an airgap install

Found that Security Onion 2.3.21 is currently installed.


Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning into 'securityonion'...

Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.

Generating new repo archive
Let's see if we need to update Security Onion.
Plenty of space for upgrading
Checking for Salt Master and Minion updates.

Performing upgrade from Security Onion 2.3.21 to Security Onion 2.3.30.

Updating dockers to 2.3.30.
Minion failed to authenticate with the master, has the minion key been accepted?
gpg: key FE507013: "Security Onion Solutions, LLC <info@securityonionsolutions.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Downloading so-acng
2.3.30: Pulling from securityonion/so-acng
Digest: sha256:b8173d97156f42ad8d4dcd0b0cb9f7cadb1ab5d264a9a082d2ffa0426ea2d359
Status: Image is up to date for quay.io/securityonion/so-acng:2.3.30
quay.io/securityonion/so-acng:2.3.30
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   543  100   543    0     0     86      0  0:00:06  0:00:06 --:--:--   114
There is a problem downloading the so-acng:2.3.30 image. Details:

gpg: Signature made Thu 18 Feb 2021 02:26:10 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"

```

**Comment by TOoSmOotH** [comment]
Can you post more information from the /root/sosetup.log?

I think this is something to do with networking/hostname.

**Reply by y0d4a** [reply]
sorry, is under  sosetup.log

but i try to do soup and get error also

`# soup
Checking to see if this is a manager.

This is a manager, We can proceed.
Checking to see if this is an airgap install

Found that Security Onion 2.3.21 is currently installed.


Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning into 'securityonion'...

Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.

Generating new repo archive
Let's see if we need to update Security Onion.
Plenty of space for upgrading
Checking for Salt Master and Minion updates.

Performing upgrade from Security Onion 2.3.21 to Security Onion 2.3.30.

Updating dockers to 2.3.30.
Minion failed to authenticate with the master, has the minion key been accepted?
gpg: key FE507013: "Security Onion Solutions, LLC <info@securityonionsolutions.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Downloading so-acng
2.3.30: Pulling from securityonion/so-acng
Digest: sha256:b8173d97156f42ad8d4dcd0b0cb9f7cadb1ab5d264a9a082d2ffa0426ea2d359
Status: Image is up to date for quay.io/securityonion/so-acng:2.3.30
quay.io/securityonion/so-acng:2.3.30
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   543  100   543    0     0     87      0  0:00:06  0:00:06 --:--:--   114
There is a problem downloading the so-acng:2.3.30 image. Details:

gpg: Signature made Thu 18 Feb 2021 02:26:10 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"
`

**Comment by y0d4a** [comment]
`

[root@SOmun ~]# cat sosetup.log

-----------------------------
 Initializing Setup
-----------------------------

2021-03-03T01:06:30Z | I | Installing as the vip user

-----------------------------
 System Characteristics
-----------------------------

2021-03-03T01:06:30Z | I | Executing command: uptime
 01:06:30 up 3 days, 14:19,  3 users,  load average: 0.00, 0.02, 0.05
2021-03-03T01:06:30Z | I | Executing command: uname -a
Linux SOmun 3.10.0-1160.11.1.el7.x86_64 #1 SMP Fri Dec 18 16:34:56 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
2021-03-03T01:06:30Z | I | Executing command: free -h
              total        used        free      shared  buff/cache   available
Mem:            94G        4.4G         69G         73M         20G         89G
Swap:            0B          0B          0B
2021-03-03T01:06:30Z | I | Executing command: lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                24
On-line CPU(s) list:   0-23
Thread(s) per core:    2
Core(s) per socket:    6
Socket(s):             2
NUMA node(s):          2
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 44
Model name:            Intel(R) Xeon(R) CPU           E5649  @ 2.53GHz
Stepping:              2
CPU MHz:               1600.000
CPU max MHz:           2533.0000
CPU min MHz:           1600.0000
BogoMIPS:              5065.30
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              256K
L3 cache:              12288K
NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,20,22
NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,21,23
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 popcnt aes lahf_lm epb ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid dtherm ida arat spec_ctrl intel_stibp flush_l1d
2021-03-03T01:06:30Z | I | Executing command: df -h
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  48G     0   48G   0% /dev
tmpfs                     48G  276K   48G   1% /dev/shm
tmpfs                     48G   74M   48G   1% /run
tmpfs                     48G     0   48G   0% /sys/fs/cgroup
/dev/mapper/system-root  838G   14G  824G   2% /
/dev/sda2                477M  163M  286M  37% /boot
/dev/mapper/nsm-nsm      1.7T  6.7G  1.7T   1% /nsm
tmpfs                    9.5G     0  9.5G   0% /run/user/1000
2021-03-03T01:06:30Z | I | Executing command: ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: enp3s0f0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc mq state DOWN group default qlen 1000
    link/ether 3c:d9:2b:07:9e:10 brd ff:ff:ff:ff:ff:ff
3: enp3s0f1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc mq state DOWN group default qlen 1000
    link/ether 3c:d9:2b:07:9e:12 brd ff:ff:ff:ff:ff:ff
4: enp4s0f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 3c:d9:2b:07:9e:14 brd ff:ff:ff:ff:ff:ff
    inet X.X.X.X/26 brd X.X.X.X scope global noprefixroute enp4s0f0
       valid_lft forever preferred_lft forever
5: enp4s0f1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc mq state DOWN group default qlen 1000
    link/ether 3c:d9:2b:07:9e:16 brd ff:ff:ff:ff:ff:ff
65: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default
    link/ether 02:42:66:56:87:8b brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/24 brd 172.17.0.255 scope global docker0
       valid_lft forever preferred_lft forever
Beginning Security Onion network install
Detecting Base OS
Loaded plugins: fastestmirror, versionlock
Existing lock /var/run/yum.pid: another copy is running as pid 13008.
Another app is currently holding the yum lock; waiting for it to exit...
  The other application is: yum
    Memory :  95 M RSS (495 MB VSZ)
    Started: Tue Mar  2 23:30:58 2021 - 1:54:13 ago
    State  : Traced/Stopped, pid: 13008


Exiting on user cancel.
Found OS: centos 7
Testing if setup is running on a cloud instance...
`

**Comment by y0d4a** [comment, marked as answer]
ok, new image 2.3.30 pass fine on both servers.
